.github/pull_request_template.md

```diff
@@ -11,6 +11,8 @@ _Describe any minor or "drive-by" changes here._
 
 _An explanation of how the changes were tested or an explanation as to why they don't need to be._
 
+_Does this need to be tested by QA in the next release cycle? If so please give a brief explanation of how to test these changes._
```

Comment:
Can we put this under it's own header? I'm afraid it will be missed if included within `Description`?

Reply:
Moved to its own section in commit [a78b97b](https://github.com/celo-org/wallet/pull/281/commits/a78b97bbf03aff485624e328583c0264e3456c8c)